### PR TITLE
[grafana] New plugin for visualizing metrics

### PR DESCRIPTION
**Pull request by pbandark**
Plugin to capture grafana configuration, logs, and command
outputs.
- configuration:
  - /etc/sysconfig/grafana-server
  - /etc/grafana/
- logs:
  - /var/log/grafana/
- command outputs:
  - grafana-cli  plugins ls
  - grafana-cli plugins list-remote
  - grafana-cli -v
  - grafana-server -v

Closes-Bug: #860

Signed-off-by: Pratik Bandarkar pbandark@redhat.com

### CODE REVIEW

File: sos/plugins/grafana.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    """Grafana
+    """
+    plugin_name = "grafana"
+    profiles = ('services',)
+
+    def setup(self):
+        self.limit = self.get_option("log_size")
+        if self.get_option("all_logs"):
+            self.add_copy_spec_limit("/var/log/grafana/",
+                                     sizelimit=self.limit)
```
Comment (bmr-cymru) [inline sos/plugins/grafana.py]: If you're doing `all_logs` there should be no size limits applied.. (it literally means "all logs").

**Comment by pbandark** [comment]
@bmr-cymru updated the code as per suggested change.

**Comment by pbandark** [comment]
@bmr-cymru any further suggestion on the commit?

**Comment by bmr-cymru** [comment]
You seem to have replaced the original commit to add the plugin with a new one (which also adds the plugin), but which has a subject of `Dont limit size while capturing all_logs `.

That doesn't seem right.

Also, you need to follow the [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines) for commit message formatting - it should be something like `[grafana] add new plugin`.

**Comment by bmr-cymru** [comment]
Closed in commit 5c2332c

**Pull request closed without merging**